app/src/main/java/com/example/githubapp/BindingAdapter.kt

```diff
@@ -15,7 +15,11 @@ fun hideIfNull(view: View, string: String?) {
 
 @BindingAdapter("app:scr")
 fun uploadImage(view: ImageView, string: String?) {
-    Picasso.with(view.context).load(string).into(view)
+    Picasso.with(view.context)
+        .load(string)
+        .placeholder(R.drawable.placeholder)
+        .transform(CircularTransformation())
```

Comment:
With  such transformation all images in the app will be round. It makes sense to add circular transformation as an option.